docs/contributing/core/index.md

```diff
@@ -0,0 +1,268 @@
+    "property=og:title": "Contribute to Plone 6 Core"
+    "keywords": "Plone, Plone Contributor Agreement, License"
+---
+
+# Contribute to Plone 6 core
+
+This guide describes the process of how to contribute to, and develop in, Plone core.
+It expands upon {doc}`/contributing/index`.
+
+This guide assumes that you have basic knowledge of how to use git and GitHub.
+If you have never contributed to Plone, or you lack basic knowledge of how to use git and GitHub, you should first read {doc}`/contributing/first-time` for more information.
+
+```{important}
+You must {ref}`contributing-sign-and-return-the-plone-contributor-agreement-label` before your contribution can be accepted.
+```
+
+
+## Version support policy
+
+Before you contribute to Plone core, check the [version support policy](https://plone.org/download/release-schedule) to see which versions of Plone are currently supported.
+
+
+## Pre-requisites
+
+It is beyond the scope of this documentation to provide installation instructions for all pre-requisites for your operating system.
+However, the following links and sections below may be helpful.
+
+```{include} ../../volto/contributing/install-operating-system.md
+```
+
+-   Python {SUPPORTED_PYTHON_VERSIONS}
+-   {term}`GNU make`
+-   {term}`Git`
+-   [libxml2 and libxslt](https://gitlab.gnome.org/GNOME/libxslt/-/releases), including development headers
+-   [GNU Compiler Collection (GCC)](https://gcc.gnu.org/) to compile {term}`ZODB`, {term}`Zope`, and {term}`lxml`
+
+
+### Python
+
+Installing Python is beyond the scope of this documentation.
+However, it is recommended to use a Python version manager, {term}`pyenv` that allows you to install multiple versions of Python on your development environment without destroying your system's Python.
+Plone requires Python version {SUPPORTED_PYTHON_VERSIONS}.
+
+
+### Make
+
+```{include} ../../volto/contributing/install-make.md
+```
+
+
+### Git
+
+```{include} ../../volto/contributing/install-git.md
+```
+
+### libxml2, libxslt, and GCC
+
+Consult the resources at the links above for installation instructions.
+Package managers may facilitate installation.
+
+
+## Install Plone core for development
+
+To set up a Plone 6 development environment, change your working directory to wherever you place your projects, and clone https://github.com/plone/buildout.coredev.
+
+```shell
+cd [MY_PROJECTS]
+git clone https://github.com/plone/buildout.coredev
+cd buildout.coredev
+```
+
+Now run the script to install Plone 6.
+
+```shell
+./bootstrap.sh
+```
+
+If you run into issues in this process, see {doc}`troubleshooting`.
+
+This will run for a long time if it's your first pull (approximately 10-20 minutes, depending on network speed and your hardware).
+
+Once that's done, you can start an instance of Plone with the following command.
+
+```shell
+./bin/instance fg
+```
+
+To visit your Plone instance, you can open the link http://0.0.0.0:8080 in a web browser.
```

Comment:
Just saw this come through the daily flood of GitHub emails:

https://github.com/plone/plone.volto/issues/155

@ericof @davisagli @sneridagh is `plone.volto`'s `README.md` the official docs for how to install Volto using buildout? The missing link, if you will.

Reply:
@stevepiercy The change you made is good enough for now.

We could add a note to https://6.docs.plone.org/volto/contributing/developing-core.html#start-the-backend-and-volto explaining that you also have the option of installing and starting the Plone backend in some other way -- such as with the coredev buildout instead of `make start-backend-docker`. As long as it's running on port 8080, the default Volto settings will use it.

If you're using buildout and you're going to use the Plone backend with Volto, then you need to choose "Create a new Plone site" instead of "Create Classic UI Plone site" on the screen that looks like this:
<img width="846" alt="Screenshot 2024-06-25 at 9 46 36 PM" src="https://github.com/plone/documentation/assets/49014/3fb22b7f-0aa8-4fd8-97bb-b3389f96ea5b">
This is what installs plone.volto which applies some extra configuration needed for Volto.

This screen will be replaced with plone.distribution's install page in Plone 6.1. If you installed Plone from the cookiecutter instead of buildout, then the make commands take care of creating the Plone site for you so you don't need to use this screen.